### PR TITLE
refactored some python targets in Rakefile for use with crazyfun

### PR DESCRIPTION
**Pull request by joshmgrant**
This is my first set of (small) changes to update some of the python targets for crazyfun. I've moved some logic directly from the Rakefile to related files in py/.

### CODE REVIEW

**Comment by ddavison** [comment]
have you signed the CLA? (see [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla))

**Comment by joshmgrant** [comment]
@ddavison I have signed the CLA about 15 min ago.

File: rake-tasks/crazy_fun/mappings/python.rb [hunk 1]
```diff
@@ -213,5 +217,45 @@ def handle(fun, dir, args)
...
+          cp "cpp/prebuilt/i386/libnoblur.so", x86+"x_ignore_nofocus.so", :verbose => true
+          cp "cpp/prebuilt/amd64/libnoblur64.so", amd64+"x_ignore_nofocus.so", :verbose => true
+		 
+          cp firefox_build_dir + "webdriver.xpi" , firefox_py_home, :verbose => true
+          cp firefox_build_dir + "webdriver_prefs.json" , firefox_py_home, :verbose => true
+		end
+	  end
+	end
+	
+
```
Comment (andreastt) [inline rake-tasks/crazy_fun/mappings/python.rb]: Remove superfluous new line

File: rake-tasks/crazy_fun/mappings/python.rb [hunk 2]
```diff
@@ -213,5 +217,45 @@ def handle(fun, dir, args)
     end
   end
 
+	class Install < Tasks
```
Comment (andreastt) [inline rake-tasks/crazy_fun/mappings/python.rb]: Follow PEP8 in this code

File: py/build.desc [hunk 3]
```diff
@@ -89,3 +89,11 @@ py_env(
   ],
   dest = "build/python"
 )
+
+py_install(
+	name = "install"
+)
```
Comment (andreastt) [inline py/build.desc]: Please use two space indent, as this is actually just Ruby code.  Alternatively put it on one line.

**Comment by andreastt** [comment]
Thanks for the patch, it looks really good!  Can you fix up the nitpicks I made about code style so we can merge it?

**Comment by joshmgrant** [comment]
@andreastt Will do. I just found out about Ruby code styling the other day (never used it) so I'll clean up some of the changes.

File: rake-tasks/crazy_fun/mappings/python.rb [hunk 4]
```diff
@@ -12,6 +12,10 @@ def add_all(fun)
     fun.add_mapping("py_env", Python::VirtualEnv.new)
 
     fun.add_mapping("py_docs", Python::GenerateDocs.new)
+	
+	fun.add_mapping("py_install", Python::Install.new)
+	
+	fun.add_mapping("py_prep", Python::Prep.new)
```
Comment (andreastt) [inline rake-tasks/crazy_fun/mappings/python.rb]: Too much indentation here.  Should match the lines above, which have 4 spaces.

File: rake-tasks/crazy_fun/mappings/python.rb [hunk 5]
```diff
@@ -213,5 +217,43 @@ def handle(fun, dir, args)
...
+	    firefox_py_home = "py/selenium/webdriver/firefox/"
+	    firefox_build_dir = 'build/javascript/firefox-driver/'
+	    x86 = firefox_py_home + "x86/"
+	    amd64 = firefox_py_home + "amd64/"
+		
+	    if (windows?) then
+		  firefox_build_dir = firefox_build_dir.gsub(/\//, "\\")
+		  firefox_py_home = firefox_py_home .gsub(/\//, "\\")
+		  x86 = x86.gsub(/\//,"\\")
+		  amd64 = amd64.gsub(/\//,"\\")
```
Comment (andreastt) [inline rake-tasks/crazy_fun/mappings/python.rb]: Indentation is too much here.

File: Rakefile [hunk 6]
```diff
@@ -385,37 +385,11 @@ task :javadocs => [:common, :firefox, :htmlunit, :ie, :remote, :support, :chrome
...
-        mkdir_p amd64 unless File.exists?(amd64)
-
-        cp "cpp/prebuilt/i386/libnoblur.so", x86+"x_ignore_nofocus.so", :verbose => true
-        cp "cpp/prebuilt/amd64/libnoblur64.so", amd64+"x_ignore_nofocus.so", :verbose => true
-
-        cp firefox_build_dir + "webdriver.xpi" , firefox_py_home, :verbose => true
-        cp firefox_build_dir + "webdriver_prefs.json" , firefox_py_home, :verbose => true
-    end
-end
+task :py_prep_for_install_release => ["//javascript/firefox-driver:webdriver", :chrome, "//javascript/firefox-driver:webdriver_prefs", "//py:prep"]
```
Comment (andreastt) [inline Rakefile]: Break array into multiple lines because it's very long.

**Comment by andreastt** [comment]
Thanks! Squashed, rebase, and integrated in 1eaa4d921b818798f6379f1d948d3a24a321d054.

Can you please update your git username to your full name?  It currently just reads “josh”.

**Pull request closed without merging**